### PR TITLE
Moki ruby action

### PR DESCRIPTION
**Pull request by leeacto**
@umtrey

### CODE REVIEW

File: lib/moki_ruby/action.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Action
+  attr_accessor :id, :last_seen, :action, :status, :client_name, :item_name, :third_party_user, :payload
```
Comment (umtrey) [inline lib/moki_ruby/action.rb]: based on the spec, there can be a `note` attribute as well returned

**Comment by umtrey** [comment]
love it, thank you

**Comment by umtrey** [comment]
actually, can you squash first

**Comment by leeacto** [comment]
+1 from @umtrey

**Pull request merged**